### PR TITLE
feat(ci): lint PR title

### PR DESCRIPTION
**Pull request by chrisgacsal**
## Description

Ensure that Pull Request title is aligned with [Conventional Commit Spec](https://www.conventionalcommits.org/en/v1.0.0/).
This allows us to generate well-formatted changelog in release notes.

## Type of Change

[ ] Bug Fix  
[x] New Feature  
[ ] Breaking Change  
[ ] Refactor  
[ ] Documentation  
[ ] Other (please describe)  

## Checklist

- [x] I have read the [contributing guidelines](https://github.com/openclarity/vmclarity/blob/main/CONTRIBUTING.md)
- [x] Existing issues have been referenced (where applicable)
- [x] I have verified this change is not present in other open pull requests
- [x] Functionality is documented
- [x] All code style checks pass
- [x] New code contribution is covered by automated tests
- [x] All new and existing tests pass

### CODE REVIEW

File: .github/workflows/semantic-pr.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            ci
+            chore
+            revert
+            BREAKING
+
+          scopes: |
+            api
+            cli
+            orchestrator
+            provider
```
Comment (paralta) [inline .github/workflows/semantic-pr.yml:35]: Can we also add `ui` and `uibackend` here? To include all service names

Comment (paralta) [reply .github/workflows/semantic-pr.yml:35]: Will this allow an empty scope?

Comment (chrisgacsal) [reply .github/workflows/semantic-pr.yml:50]: > Will this allow an empty scope?

I have not tested, so I've removed the empty lines to be on the safe side. Although it seems the scopes and types are regex patterns wrapped in `^ $` which will make an empty line a non-matching pattern according to the [docs](https://github.com/amannn/action-semantic-pull-request#configuration).

Comment (paralta) [reply .github/workflows/semantic-pr.yml:50]: There is a `requireScope: true` in the link you attached! I think that would be useful for example when the type is `docs `

**Review by paralta — APPROVED** [review]

File: .github/workflows/semantic-pr.yml (hunk 1 above)
Comment (chrisgacsal) [reply .github/workflows/semantic-pr.yml:50]: Afaik `requireScope: false` is the default.

**Review by paralta — APPROVED** [review]

**Review by paralta — APPROVED** [review]
LGTM 👍

**Review by lgecse — APPROVED** [review]
👍 LGTM!

**Pull request merged**